### DISCUSSION TITLE
Native support for ARM Architecture.

### DISCUSSION THREAD

**Discussion by Vretu-Dev** [lavalink-devs/Lavalink#647] [Q&A]
A lot of people use ARM-based processors, the version compiled by Cog-Creators doesn't work quite right. https://github.com/Cog-Creators/Lavalink-Jars/releases.
It would be useful to have a natively supporting version of lavalink running on ARM

**Comment by topi314** [comment]
Lavaplayer is planning to support arm & many other architectures with the next version(`1.3.98`)

You can look forward to this!

**Comment by topi314** [comment]
this change has finally been released with https://github.com/freyacodes/Lavalink/releases/tag/3.5-rc2
